### PR TITLE
Add known limitations, mention dart2aot

### LINKED ISSUE #2068: Clarify dart2native platform support
The dart2native page should incorporate the information in the **Known limitations** section of the [2.6 announcement](https://medium.com/dartlang/dart2native-a76c815e6baf).

/cc @mit-mit

### PR DESCRIPTION
**Pull request by kwalrath**
Fixes #2068
Staged: https://sz-www-1.firebaseapp.com/tools/dart2native

### CODE REVIEW

File: src/tools/dart2native.md [hunk 1]
```diff
@@ -56,6 +56,39 @@ You can then run the app using the [`dartaotruntime` command][]:
 $ dartaotruntime bin/main.aot
 ```
 
+## Known limitations
+
+The initial (Dart 2.6) version of `dart2native` some known limitations:
```
Comment (chalin) [inline src/tools/dart2native.md]: Verb missing?

Comment (kwalrath) [reply src/tools/dart2native.md]: Oops! fixed

**Comment by kwalrath** [comment]
Since we redirect dart2aot here, we should mention it in the page.

**Comment by kwalrath** [comment]
I'm going to commit this TBR.

File: src/tools/dart2native.md [hunk 2]
```diff
@@ -24,6 +24,13 @@ An **AOT snapshot** doesn't include the Dart runtime.
 Consider using snapshots if you're distributing multiple programs
 and disk space is limited.
 
+{{site.alert.note}}
```
Comment (mit-mit) [inline src/tools/dart2native.md]: I think I'd prefer this at the bottom of the page. It's really only applicable to a very, very small user base, so should not be placed so prominently

File: src/tools/dart2native.md [hunk 3]
```diff
@@ -56,6 +63,39 @@ You can then run the app using the [`dartaotruntime` command][]:
...
+  A workaround is to use a CI (continuous integration) provider
+  that supports all three operating systems.
+
+No signing support ([issue 39106][])
+: The format of the executables isn’t compatible with
+  standard signing tools such as codesign and SignTool.
+
+No support for dart:mirrors and dart:developer
+: The code compiled by `dart2native` can use all of the other libraries
+  that the Dart VM supports.
```
Comment (mit-mit) [inline src/tools/dart2native.md]: On the page linked to here, the term 'Dart VM' isn't really used

Comment (kwalrath) [reply src/tools/dart2native.md]: That page should explain that JIT means — it's very unclear. I think we meant to do that but forgot.

File: src/tools/dart2native.md (hunk 2 above)
Comment (kwalrath) [reply src/tools/dart2native.md]: I'll put that in a new PR.

**Pull request merged**